sso/sso72-x509-https.json

```diff
@@ -0,0 +1,521 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "annotations": {
+            "iconClass" : "icon-sso",
+            "tags" : "sso,keycloak,jboss,hidden",
+            "version": "1.4.10",
+            "openshift.io/display-name": "Single Sign-On 7.2",
+            "openshift.io/provider-display-name": "Red Hat, Inc.",
+            "description": "An example RH-SSO 7 application. For more information about using this template, see https://github.com/jboss-openshift/application-templates.",
+            "template.openshift.io/long-description": "This template defines resources needed to develop Red Hat Single Sign-On 7.2 server based deployment.",
+            "template.openshift.io/documentation-url": "https://access.redhat.com/documentation/en/red-hat-single-sign-on/",
+            "template.openshift.io/support-url": "https://access.redhat.com"
+        },
+        "name": "sso72-x509-https"
+    },
+    "labels": {
+        "template": "sso72-x509-https",
+        "xpaas": "1.4.10"
+    },
+    "message": "A new RH-SSO service has been created in your project. The admin username/password for accessing the master realm via the RH-SSO console is ${SSO_ADMIN_USERNAME}/${SSO_ADMIN_PASSWORD}. The HTTPS keystore used for serving secure content, the JGroups keystore used for securing JGroups communications, and server truststore used for securing RH-SSO requests were automatically created via OpenShift's service serving x509 certificate secrets.",
+    "parameters": [
+        {
+            "displayName": "Application Name",
+            "description": "The name for the application.",
+            "name": "APPLICATION_NAME",
+            "value": "sso",
+            "required": true
+        },
+        {
+            "displayName": "Server HTTPS Keystore Password",
+            "description": "The password for the HTTPS keystore.",
+            "name": "HTTPS_PASSWORD",
```

Comment:
why is this needed in this template?

Reply:
Just this way seemed more elegant to me, than to do something like:
```
HTTPS_PASSWORD=$(openssl rand -base64 32)
```
or
```
PASSWORD=$(openssl rand -base64 32)
HTTPS_PASSWORD=${PASSWORD}
```
directly in the ```openshift-x509.sh``` script (like the user having the ability to see the password, seemed more transparent way to me).

But if it's not needed (the user doesn't need to be aware of the various passwords), I can use the ```openssl rand``` approach above (remove all three of them).